cpp/include/cudf/strings/convert/convert_integers.hpp

```diff
@@ -69,5 +69,32 @@ std::unique_ptr<column> to_integers( strings_column_view const& strings,
 std::unique_ptr<column> from_integers( column_view const& integers,
                                        rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource());
 
+/**
+ * @brief Returns a new integer numeric column parsing hexidecimal values from the
```

Comment:
```suggestion
 * @brief Returns a new integer numeric column parsing hexadecimal values from the
```

Reply:
Well, that is just embarrassing. Why did I think it was spelled 'hexidecimal'?